samples/host_exerciser/host_exerciser.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
```

Comment:
Why do we need this file?

Reply:
fixed

Reply:
Multi PF & VF test tool, used to test HE-LPBK and HE-ME and HSSI